ztp/source-crs/ClusterLogForwarder.yaml

```diff
@@ -1,13 +1,18 @@
-apiVersion: "observability.openshift.io/v1"
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
   name: instance
   namespace: openshift-logging
   annotations:
     ran.openshift.io/ztp-deploy-wave: "10"
 spec:
-# outputs: $outputs
-# pipelines: $pipelines
+  outputs:
```

Comment:
Looks like this is adding something to the source CRs should this be?

Reply:
According to the CRD, these 2 fields are required anyway, so having a placeholder instead of a commented-out section is identical... It must be overridden by the end-user either way.

Plus it makes it easier to generate/validate the CRD from the reference.

Reply:
- https://github.com/openshift/cluster-logging-operator/blob/release-6.0/api/observability/v1/clusterlogforwarder_types.go#L49-L55
- https://github.com/openshift/cluster-logging-operator/blob/release-6.0/api/observability/v1/clusterlogforwarder_types.go#L67-L73